packages/compiler-cli/src/ngtsc/partial_evaluator/test/utils.ts

```diff
@@ -16,17 +16,20 @@ import {getDeclaration, makeProgram} from '../../testing';
 import {ForeignFunctionResolver, PartialEvaluator} from '../src/interface';
 import {ResolvedValue} from '../src/result';
 
-export function makeExpression(code: string, expr: string, supportingFiles: TestFile[] = []): {
+export function makeExpression(
+    code: string, expr: string, supportingFiles: TestFile[] = [], checkForErrors?: boolean): {
   expression: ts.Expression,
   host: ts.CompilerHost,
   checker: ts.TypeChecker,
   program: ts.Program,
   options: ts.CompilerOptions
 } {
-  const {program, options, host} = makeProgram([
-    {name: absoluteFrom('/entry.ts'), contents: `${code}; const target$ = ${expr};`},
-    ...supportingFiles
-  ]);
+  const {program, options, host} = makeProgram(
+      [
+        {name: absoluteFrom('/entry.ts'), contents: `${code}; const target$ = ${expr};`},
+        ...supportingFiles
+      ],
+      undefined, undefined, checkForErrors);
```

Comment:
For readability, I tend to comment each primitive/unspecified value with its argument name:

```suggestion
      /* options */ undefined, /* host */ undefined, checkForErrors);
```

Reply:
Sure, sounds good to me 👍